src/encoded/static/components/navigation/components/SearchBar.js

```diff
@@ -5,8 +5,9 @@ import PropTypes from 'prop-types';
 import url from 'url';
 import memoize from 'memoize-one';
 import _ from 'underscore';
-import { DropdownItem, DropdownButton } from 'react-bootstrap';
-import { Fade } from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/Fade';
+import DropdownItem from 'react-bootstrap/es/DropdownItem';
+import DropdownButton from 'react-bootstrap/es/DropdownButton';
+import Fade from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/Fade';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Fade' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4527127)